classes/User/User.mjs

```diff
@@ -18,6 +18,26 @@ export default class User {
     return this
   }
 
+  async updateProfile(data) {
+    if(!data) throw new Error("No data provided")
+
+    const existingUser = await database.findOne({ email: data.email }, "users")
+    if (existingUser && existingUser.email === data.email) {
```

Comment:
I understand the concern over collision. If we are validating the input, we should find a good conflict HTTP Error code. Also, the user should have a unique `_sub`. 

*However* this is just the profile, so none of that matters. Only the `profile` property should be touched.